sdk/subscription/arm-subscriptions/CHANGELOG.md

```diff
@@ -33,6 +33,22 @@
   - Removed Type Alias SpendingLimit
   - Removed Type Alias SubscriptionState
 
+If you need to query subscriptions, please switch to use the [`@azure/arm-resources-subscriptions`](https://www.npmjs.com/package/@azure/arm-resources-subscriptions) package instead. The API similarity should mean this is generally a drop-in replacement for subscription iteration:
```

Comment:
```suggestion
Operation groups SubscriptionOperations, Subscriptions and Tenants have been removed since 6.0.0. If you need to query these operation groups, please switch to use the [`@azure/arm-resources-subscriptions`](https://www.npmjs.com/package/@azure/arm-resources-subscriptions) package instead. The API similarity should mean this is generally a drop-in replacement for subscription iteration:
```